masterlist.yaml

```
@@ -8028,6 +8028,30 @@ plugins:
         util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 3
 
+  - name: 'Orden des Drachen.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/45258/'
+        name: 'Order of the Dragon EV'
+      - link: 'https://www.nexusmods.com/oblivion/mods/46786/'
+        name: 'Orden des Drachen'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Wizard''s Tower'
+          - '[Patch Orden des Drachen - Frostcrag Spire](https://www.nexusmods.com/oblivion/mods/38760/)'
+        condition: 'not active("Patch - Orden des Drachen - Frostcrag.esp")'
```

The patch message should only be displayed if Wizard's Tower is in use.

```suggestion
        condition: 'active("DLCFrostcrag.esp") and not active("Patch - Orden des Drachen - Frostcrag.esp")'
```